website/docs/contributing/ADRs.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architectural Decision Records
+---
+
+## Introduction {#introduction}
+
+Architectural decision records are a record of design decisions we have made in the past because we belived they would help our code quality over time. Any ADR can be challenged, but two conditions must be met to change an ADR: (1) The proposed solution must provide a tangible benefit in terms of code quality, and (2) the proposed solutions benefit must be worth retroactively refitted to the entire codebase. One such example is the decision to re-write unleash to typescript.
+
+## Crosscutting ADRs
```

Comment:
I wanna say this should be with a hyphen instead (i.e. "cross-cutting"), but maybe we can make it clearer by changing the word and/or by adding a description:

```suggestion
## Overarching ADRs

These ADRs describe decisions that concern the entire codebase. They apply to back-end code, front-end code, and code that doesn't neatly fit into either of those categories.
```